src/Pages/MyPin.js

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import styled from "styled-components";
+import { FiShare } from "react-icons/fi";
+import { BsThreeDots, BsArrowUpRight } from "react-icons/bs";
+
+
+
+class MyPin extends React.Component {
+
+    
+ 
+  render() {
+    const { data, componentKey } = this.props;
+    const srcLink = data.pinLink.slice(0,15) + "..."
+
+    return (
+      <PinContainer key={componentKey}>
+        <ImgContainer>
+          <SaveBtn>
+            <span>저장</span>
+          </SaveBtn>
+         {data.pinLink ?  <LinkContainer>
+            <Arrow />
+    <span style={{ fontSize: 12, fontWeight: 700 }}>{srcLink}</span>
+          </LinkContainer> : ""}
```

Comment:
1.
```suggestion
         {data.pinLink && <LinkContainer>
            <Arrow />
    <span style={{ fontSize: 12, fontWeight: 700 }}>{srcLink}</span>
          </LinkContainer>}
```

false의 경우가 없는 경우 && 연산자를 써주세요.

2. inline 스타일 제거해주세요...

Reply:
반영했습니다